db/seeds/002_spree.rb

```diff
@@ -0,0 +1,78 @@
+Zabka_Winogrady = Spree::Vendor.create(
+    name: "Zabka Winogrady",
+    about_us: "Winogrady 61",
+    postcode: "61-659"
```

Comment:
this looks hacky and we spoke about it,
The vendor has about_us & contact_us fields, what is the purpose of those fields?
In a naive implementation, we can assume that `contact_us` could be the address e.g.
`contact_us = "Winogrady 61, 61-659 Poznań"` so there is no need to have postcode.

On the other hand, there is ready to use: https://github.com/spree/spree/blob/main/core/app/models/spree/address.rb co maybe this is a way when we like to handle addresses/location on vendor model.